Packs/NistNVD/Integrations/NistNVD/NistNVD.py

```diff
@@ -0,0 +1,322 @@
+# import datetime
+import sys
+from datetime import datetime, timedelta
+
+import demistomock as demisto  # noqa: F401
+import requests
+from CommonServerPython import *  # noqa: F401
+# from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+# !/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# disable insecure warnings
+# requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+''' CONSTANTS '''
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+VERIFY_SSL = not demisto.params().get('insecure', False)
+
+if not demisto.params().get('proxy', False):
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
```

Comment:
You can use the `handle_proxy` method instead, it's the more common way to do that.

Reply:
Thanks @teizenman I didn't know this function.

Reply:
Actually, you can just call the `handle_proxy` method in the `main` function, this will handle everything you need. You wouldn't need the `if` statement.